### PR TITLE
Check API version

### PR DESCRIPTION
**Pull request by RagnarR**
* Change compileSdkVersion, minSdkVersion and targetSdkVersion at 10.
* Fix code with API 10.

### CODE REVIEW

File: library.interstitial/src/main/res/layout/pubnative_interstitial.xml [hunk 1]
```diff
@@ -62,15 +62,13 @@
...
             android:id="@+id/pubnative_interstitial_rating"
-            style="?android:attr/ratingBarStyleSmall"
+            style="@style/ratingBarStyle"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:layout_alignBottom="@id/pn_interstitial_icon"
             android:layout_marginBottom="5dp"
             android:layout_toRightOf="@id/pn_interstitial_icon"
-            android:isIndicator="true"
-            android:progressBackgroundTint="@color/pubnative_interstitial_rating_stars_empty"
```
Comment (anshumansworld) [inline library.interstitial/src/main/res/layout/pubnative_interstitial.xml]: We can remove this color from `colors.xml`

File: library.interstitial/src/main/res/layout/pubnative_interstitial.xml [hunk 2]
```diff
@@ -62,15 +62,13 @@
...
-            android:progressTint="@color/pubnative_interstitial_rating_stars_fill"/>
```
Comment (anshumansworld) [inline library.interstitial/src/main/res/layout/pubnative_interstitial.xml]: same as above

**Comment by anshumansworld** [comment]
except those two comments, it looks good to me. :+1:

File: library.interstitial/src/main/res/layout/pubnative_interstitial.xml (hunk 1 above)
Comment (RagnarR) [inline library.interstitial/src/main/res/layout/pubnative_interstitial.xml]: 👍

File: library.interstitial/src/main/res/layout/pubnative_interstitial.xml (hunk 2 above)
Comment (RagnarR) [inline library.interstitial/src/main/res/layout/pubnative_interstitial.xml]: 👍

**Pull request merged**